index.js

```diff
@@ -66,7 +66,11 @@ exports.getTestsFromArgs = function (testType, onFile, args = {}) {
       return testName !== args.test
     }
   }
-  return getTests(testType, onFile, fileFilter, skipFn)
+  if (args.testsDir) {
```

Comment:
Can you stay in the logic used for the other arguments for calling ``getTests`` instead of this call switch, e.g.

```
let testDir, fileFilter, excludeDir, skipFn

if (testType === 'VMTests') {
    skipFn = (name) => {
      return skipTest(name, args.skipVM)
    }
  }
  
  if (args.dir) {
    testDir = args.dir
  }
```

Not sure if this is the most elegant way of doing this but at least then it's consistent.

Reply:
Sure, this is reasonable.